styles/globals.css

```diff
@@ -45,6 +45,15 @@ code.language-unknown::after {
   }
 }
 
+@media screen and (max-width:1000px){
+  .h-screen{
+    height:60vh !important;
```

Comment:
Hmm not sure about the hardcoded values and the `!important` - let me do some research

Reply:
Yeah, sure